### PR TITLE
Added BokehRenderer.app method to create bokeh apps in scripts and notebooks

### PR DESCRIPTION
**Pull request by philippjfr**
Adds convenience method to create bokeh Apps from any holoviews object. In a script it simply returns the ``doc`` but when used with notebook option creates an ``Application`` instance which is displayed and returned.

### CODE REVIEW

**Comment by jlstevens** [comment]
Great!

All it needs is an example script in ``plotting/bokeh/examples/apps/apps`` and maybe a script and notebook in ``holoviews-contrib``.

**Comment by philippjfr** [comment]
>All it needs is an example script in plotting/bokeh/examples/apps/apps

I'll just change the existing examples to use this.

**Comment by jlstevens** [comment]
> I'll just change the existing examples to use this.

Makes sense.

**Comment by philippjfr** [comment]
Updated the examples, ready to merge once tests pass.

File: holoviews/plotting/bokeh/__init__.py [hunk 1]
```diff
@@ -33,6 +33,9 @@
 
 Store.renderers['bokeh'] = BokehRenderer.instance()
 
+if len(Store.renderers) == 1:
```
Comment (jlstevens) [inline holoviews/plotting/bokeh/__init__.py]: Why not check that the one available renderer is 'bokeh'?

Comment (philippjfr) [inline holoviews/plotting/bokeh/__init__.py]: Guess I could but defining that renderer is literally the line above so it has to be bokeh.

Comment (philippjfr) [inline holoviews/plotting/bokeh/__init__.py]: Is your suggestion this?

``if list(Store.renderers.keys()) == ['bokeh']``

Comment (jlstevens) [inline holoviews/plotting/bokeh/__init__.py]: I suppose that's fine.

**Comment by jlstevens** [comment]
Thanks for fixing the ``examples/apps/app`` nesting. I notice ``player.py`` was renamed but not changed..was that your intention?

File: holoviews/plotting/bokeh/__init__.py (hunk 1 above)
Comment (jlstevens) [inline holoviews/plotting/bokeh/__init__.py]: You last post appeared before my reply. Either way is fine.

Comment (philippjfr) [inline holoviews/plotting/bokeh/__init__.py]: Yes, ``player`` does not use ``BokehRenderer.app`` it's an example of creating a custom bokeh layout with custom widgets.

**Comment by philippjfr** [comment]
Yes, player does not use BokehRenderer.app it's an example of creating a custom bokeh layout with custom widgets.

File: holoviews/plotting/bokeh/__init__.py (hunk 1 above)
Comment (jlstevens) [inline holoviews/plotting/bokeh/__init__.py]: In that case, I would mention that in a script-level docstring at the top of the file. Same for the crossfilter app. It only needs to be one line explaining what these examples are supposed to show.

**Comment by jlstevens** [comment]
Looks good. Merging.

**Pull request merged**